### PR TITLE
[14.0][MIG] purchase_force_invoiced: Migration to 14.0

### PR DESCRIPTION
**Pull request by Reyes4711-S73**
Standard Migration to 14.0

### CODE REVIEW

File: purchase_force_invoiced/model/purchase.py [hunk 1]
```diff
@@ -23,15 +23,3 @@ def _get_invoiced(self):
             lambda po: po.force_invoiced and po.invoice_status == "to invoice"
         ):
             order.invoice_status = "invoiced"
-
-
-class PurchaseOrderLine(models.Model):
-    _inherit = "purchase.order.line"
-
-    def _prepare_account_move_line(self, line):
```
Comment (kittiu) [inline purchase_force_invoiced/model/purchase.py]: Need to remove this function? It does make sense to have it, isn't it?

Comment (Reyes4711-S73) [reply purchase_force_invoiced/model/purchase.py]: This function is removed due to in 14.0 version, the field doesn't work with this function

File: purchase_force_invoiced/model/purchase.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    _inherit = "purchase.order"
+
+    force_invoiced = fields.Boolean(
+        string="Force invoiced",
+        readonly=True,
+        states={"done": [("readonly", False)]},
+        copy=False,
+        help="When you set this field, the purchase order will be "
+        "considered as fully billed, even when there may be ordered "
+        "or delivered quantities pending to bill.",
```
Comment (alexis-via) [inline purchase_force_invoiced/model/purchase.py]:
```suggestion
        "or delivered quantities pending to bill. To use this field, "
        "the order must be in 'Locked' state.",
```

**Review by alexis-via — APPROVED** [review]
Approved, with 1 suggestion to improve help message.

**Comment by Reyes4711-S73** [comment]
@alexis-via  Suggestion added

**Review by olgamarcocb — APPROVED** [review]
Locally tested 👍

**Comment by alexis-via** [comment]
Let's merge ! Someone with the power to merge is around?

**Review by rvalyi — APPROVED** [review]

**Comment by rvalyi** [comment]
the error was with purchase_quick. Trying to merge again...

**Comment by rvalyi** [comment]
We still have a problem with the purchase_quick tests and it seems unrelated to this module: https://app.travis-ci.com/github/OCA/purchase-workflow/jobs/555270394


@sebastienbeau @mourad-ehm @kevinkhao @PierrickBrun as you are the authors of purchase_quick any chance you could check what is going on? It seems the failing tests block the whole purchase-workflow merges on v14...
See another example here https://app.travis-ci.com/github/OCA/purchase-workflow/jobs/555198721

**Review by etobella — APPROVED** [review]

**Comment by rvalyi** [comment]
OK, the tests in purchase_quick were fixed here https://github.com/OCA/purchase-workflow/commit/5bc476cccd676de2fb2ad6a497178e5b8c848337 so we can now merge this one...

**Pull request merged**